runtime/common/src/crowdloan/mod.rs

```diff
@@ -51,6 +51,8 @@
 
 pub mod migration;
 
+// TODO: Expose the total amount held.
```

Comment:
Wouldn't [ReservedAmount](https://github.com/paritytech/polkadot/blob/release-v0.9.32/runtime/common/src/auctions.rs#L190L195) from the Auctions pallet capture everything in crowdloans and self-funded bids?

Reply:
Really, how? 

- Auction bids are reserved: https://github.com/paritytech/polkadot/blob/release-v0.9.32/runtime/common/src/auctions.rs#L480
- And vote requires free balance: https://github.com/paritytech/substrate/blob/polkadot-v0.9.32/frame/conviction-voting/src/lib.rs#L386